### DISCUSSION TITLE
feat(CardUpload): auto clear when invalid file type

### DISCUSSION THREAD

**Discussion by ham187** [dotnetcore/BootstrapBlazor#4371] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the solution you'd like

If user selects incorrect file type (.doc instead of .jpg), reset the control instead of requiring delete first

<img width="419" alt="cardupload" src="https://github.com/user-attachments/assets/31268e8f-360d-4762-9073-09f91c99e707">

Is the code okay? or is there a better way?

**Comment by bb-auto** [comment]
@ham187 Thank you for contacting us. We will give feedback later.

**Comment by ArgoZhang** [comment]
@ham187 could you provide a mini-repro-project and detailed repro steps?

**Reply by ArgoZhang** [reply]
@ham187 could you give this project a star? thanks

I'll take a look at this function. It's not easy to do it inside the component. I think the code in your screenshot is fine.
